src/hasher.rs

```diff
@@ -85,7 +86,7 @@ impl WyHash {
     #[inline]
     fn consume_bytes(&self, bytes: &[u8]) -> (u64, u64, u64) {
         let length = bytes.len();
-        if length <= 0 {
+        if length == 0 {
```

Comment:
Changed this because Clippy got mad at me

Reply:
well, that's a reason to never do that again. I'll keep that in mind for the future, lol.